etc/kayobe/ansible/pulp-host-image-upload.yml

```diff
@@ -25,6 +25,10 @@
         password: "{{ remote_pulp_password }}"
         file: "{{ found_files.files[0].path }}"
         state: present
+      register: upload_result
+      until: upload_result is success
+      retries: 3
+      delay: 5
```

Comment:
The other plays are fine because they're metadata but this one might benefit from a longer delay since it's trying to upload a relatively large artifact. Maybe a minute or two?